meta-phosphor/conf/machine/include/obmc-bsp-common.inc

```diff
@@ -13,7 +13,7 @@ UBOOT_SUFFIX ?= "bin"
 
 MACHINEOVERRIDES =. "openbmc:"
 
-IMAGE_FSTYPES += "overlay"
+MACHINE_FEATURES += "image-overlay"
```

Comment:
Could you achieve the same thing but with a different variable?  This doesn't seem like a machine feature to me.  Neither did pflash for that matter.

In general features (whether they be image features, distro features or machine features) control what packages go into an image when the image recipe is built.  This is different - this is figuring out how to package the root filesystem after all the packages have been installed into it.

In my head machine features are hardwares features a machine supports.  Like "bluetooth" or "network" or "has fans".  Then if both distro features and machine features support a given function, it is enabled and put in the image.

I am certainly open to having my interpretation adjusted...